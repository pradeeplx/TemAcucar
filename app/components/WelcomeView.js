import React from 'react'
import { View } from 'react-native'
import Colors from "../Colors"

export default WelcomeView = ({ children, style }) => (
  <View style={[{
    flex: 1,
    alignItems: 'center',
    backgroundColor: Colors.white,
  }, style]}>
    { children }
  </View>
)
