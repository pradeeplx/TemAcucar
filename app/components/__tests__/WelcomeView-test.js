import 'react-native'
import React from 'react'
import WelcomeView from '../WelcomeView'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <WelcomeView
      style={{top: 0}}
    >
      FooBar
    </WelcomeView>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
