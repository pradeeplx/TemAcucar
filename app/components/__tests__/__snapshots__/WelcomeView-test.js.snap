exports[`test renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "flex": 1,
      },
      Object {
        "top": 0,
      },
    ]
  }>
  FooBar
</View>
`;
